clang/test/CodeGenSYCL/multiple_aspect_enum.cpp

```diff
@@ -1,12 +0,0 @@
-// RUN: %clang_cc1 -internal-isystem %S/Inputs -fsycl-is-device -triple spir64-unknown-unknown -verify -emit-llvm-only %s
-
-// Tests for error diagnostics when multiple definitions of
-// [[__sycl_detail__::sycl_type(aspect)]] enums are present.
-#include "sycl.hpp"
-
-// expected-note@#AspectEnum{{previous definition is here}}
-
-// expected-error@+1{{redefinition of aspect enum}}
-enum class [[__sycl_detail__::sycl_type(aspect)]] aspect_redef {
-  imposter_value = 3
-};
```

Comment:
What happens for this testcase after your changes?

Reply:
No error or note will be emitted, I removed the [code](https://github.com/intel/llvm/pull/13194/files/2624b0f8215554857214860b9d67e8c564c34470#diff-e724febedab9c1a2832bf2056d208ff02ddcb2e6f90b5a653afc9b19ac78a5d7L5696-L5705) that would do so otherwise